tests/foreman/performance/common/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
+
+Part of functionalities of Candlepin are defined in this module
+and have utilities of single register by activation-key, single
+register and attach, single subscription deletion.
+
+"""
+
+
+from robottelo.common import conf, ssh
+
+import logging
+import time
+import requests
+
+
+class CandlepinPerfTest:
+    """Measures performance of RH Satellite 6
+
+    Candlepin Subscription functionality
+    """
+    username = conf.properties['foreman.admin.username']
+    password = conf.properties['foreman.admin.password']
+    serverhost = conf.properties['main.server.hostname']
+
+    # create logger
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
```

Comment:
As for perf-candlepin module, since it is not a test case, just a helper class, I don't think it could inherit TestCase... Thus I have to get the logger here. </br> On the other hand, I observe that modules in robottelo.cli hardly generate any output. So, in which way would u recommend me to get the logger?